docker-cpp-demos/amazonlinux-docker/README.md

```diff
@@ -0,0 +1,47 @@
+# Using Docker images for Producer SDK (CPP and GStreamer plugin):
+
+Please follow the four steps below to get the Docker image for Producer SDK (includes CPP and GStreamer plugin) and start streaming to Kinesis Video.
+
+#### Pre-requisite:
+
+This requires docker is installed in your system.
+
+Follow instructions to download and start Docker
+
+* [Docker download instructions](https://www.docker.com/community-edition#/download)
+* [Getting started with Docker](https://docs.docker.com/get-started/)
+
+#### Step 1: Build the docker image
+
+Run the following command: 
+`docker build -t kinesis-video-producer-sdk-cpp-amazon-linux.`
+
+This takes some time as it pulls all the dependencies in.
+
+#### Step 2: Find the docker image
+Run the following command to find the image id for `kinesis-video-producer-sdk-cpp-amazon-linux`:
+`docker images`
+
+
+#### Step 3: Start the container
+Run the following command to start the kinesis video sdk container. Note that this will not work in macOS
+
+`sudo docker run -it --device=/dev/video0 --device=/dev/vchiq -v /opt/vc:/opt/vc <image-id> /bin/bash`
+
+#### Step 4: Run the gstreamer sample
+
+Set these environment variables: 
+`export GST_PLUGIN_PATH=/opt/amazon-kinesis-video-streams-producer-sdk-cpp/build`
```

Comment:
I don't think exporting these envs is necessary since you already set them in the dockerfile, https://github.com/aws-samples/amazon-kinesis-video-streams-demos/pull/6/files#diff-a5d2af6f1d34eea5ca6f9fb914ff22f3R52-R53